File: stateLevel/java/integerSum.java
import java.util.Scanner;

public class integerSum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int sum = 0;

        while (true) {
            String input = scanner.nextLine();

            if (input.equalsIgnoreCase("sum")) {
                break;
            }

            try {
                int number = Integer.parseInt(input);
                sum += number;
            } catch (NumberFormatException e) {
                System.out.println("Invalid input.");
                return;
            }
        }

        System.out.println(sum);
    }
}

File: stateLevel/py/integerSum.py
def integerSum():
    numbers = []
    while True:
        try:
            inputNumber = input()
            if inputNumber == 'sum':
                break
            if inputNumber.isdigit():
                numbers.append(int(inputNumber))
            else:
                print('Invalid input.')
                return
        except ValueError:
            print('Invalid input.')
            return
    print(sum(numbers))

if __name__ == '__main__':
    integerSum()